frontend/src/v5/ui/routes/viewer/tickets/newTicket/newTicket.component.tsx

```diff
@@ -0,0 +1,165 @@
+import { useParams } from 'react-router-dom';
+import { FormProvider, useForm } from 'react-hook-form';
+import * as Yup from 'yup';
+import { CircularProgress } from '@mui/material';
+import TicketsIcon from '@mui/icons-material/FormatListBulleted';
+import { CardContainer, CardHeader } from '@/v5/ui/components/viewer/cards/card.styles';
+import { CardContent } from '@/v5/ui/components/viewer/cards/cardContent.component';
+import { CardContext } from '@components/viewer/cards/cardContext.component';
+import { Button } from '@controls/button';
+import { TicketsHooksSelectors } from '@/v5/services/selectorsHooks/ticketsSelectors.hooks';
+import { EditableTicket, NewTicket } from '@/v5/store/tickets/tickets.types';
+import { filterEmptyValues, modelIsFederation, propertiesValidator, propertyValidator } from '@/v5/store/tickets/tickets.helpers';
+import { TicketsActionsDispatchers } from '@/v5/services/actionsDispatchers/ticketsActions.dispatchers';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { BottomArea, Form, SaveButton } from './newTicket.styles';
+import { TicketForm, TITLE_INPUT_NAME } from '../ticketsForm/ticketsForm.component';
+import { TicketsCardViews } from '../tickets.constants';
+import { ViewerParams } from '../../../routes.constants';
+
+export const NewTicketCard = () => {
+	const contextValue = useContext(CardContext);
+	const templateId = contextValue.props.template._id;
+	const { teamspace, project, containerOrFederation } = useParams<ViewerParams>();
+	const isFederation = modelIsFederation(containerOrFederation);
+	const template = TicketsHooksSelectors.selectTemplateById(containerOrFederation, templateId);
+	const isLoading = !('config' in template);
+
+	const getEditableProperties = () => {
+		const NON_EDITABLE_PROPERTIES = ['Owner', 'Created at', 'Updated at'];
+		return template.properties?.filter(({ name }) => !NON_EDITABLE_PROPERTIES.includes(name));
+	};
+
+	const editableTemplate = {
+		...template,
+		properties: getEditableProperties(),
+	};
+
+	const getValidators = () => {
+		if (isLoading) return null;
+
+		const { properties, modules } = editableTemplate;
+		const validators: any = {
+			title: propertyValidator({
+				required: true,
+				type: 'longText',
+				name: TITLE_INPUT_NAME,
+			}),
+		};
+		validators.properties = propertiesValidator(properties);
+		modules.forEach((module) => {
+			validators[module.name] = propertiesValidator(module.properties);
+		});
+
+		return Yup.object().shape(validators);
+	};
+
+	const formData = useForm({
+		resolver: yupResolver(getValidators()),
+		mode: 'onChange',
+	});
+
+	const goBack = () => {
+		contextValue.setCardView(TicketsCardViews.List);
+	};
+
+	const goToTicketDetails = (ticketId) => {
+		contextValue.setCardView(TicketsCardViews.Details, { ticketId });
+	};
+
+	const changeTemplate = () => {
+		contextValue.setCardView(TicketsCardViews.Templates);
+	};
+
+	const getBaseTicket = (): EditableTicket => {
```

Comment:
This one's wrong it equating ticket with template.
I see two ways of doing this:

1 . I would take out the "defaultvalue" prop of the properties . And create a base ticket using the default value information from the template.

2. If Im not mistaken (I guess the defaultValue is set if the value is undefined) just pass an empty ticket and default value would take care of the rest? 

Also, move this one to the tickets helper